utils/emoji_utils.jsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function sortEmojis(emojiA, emojiB, searchedName) {
```

Comment:
I think this compare function could have a better name since it doesn't sort emoji.

Reply:
This is a good point. I'll rename the function which will also resolve your other comment about naming. Thanks for the input!